app/models/state_file1099_g.rb

```diff
@@ -38,6 +38,7 @@ class StateFile1099G < ApplicationRecord
   enum recipient: { unfilled: 0, primary: 1, spouse: 2 }, _prefix: :recipient
 
   validates_inclusion_of :had_box_11, in: ['yes', 'no'], message: ->(_object, _data) { I18n.t("errors.messages.blank") }
+  validates :address_confirmation, inclusion: { in: %w[yes no], message: ->(_object, _data) { I18n.t("errors.messages.blank") } }, if: :had_box_11_yes?
```

Comment:
this does the same thing without having to specify input types or error message, right?
```suggestion
  validates :address_confirmation, presence: true, if: :had_box_11_yes?
```

Reply:
That is what I initially did but the error message does not show up that way